SCSH_3C_iOS/Persistence/CoreData/Distributor/DistributorLikeCoreDataDao.swift

```diff
@@ -0,0 +1,117 @@
+//
+//  DistributorLikeCoreDataDao.swift
+//  Persistence
+//
+//  Created by 辜敬閎 on 2023/12/4.
+//
+
+import CoreData
+
+protocol DistributorLikeCoreDataDaoType {
+    func get(id: Int64?) throws -> [Distributor_Like]
```

Comment:
I am not sure if it is best to pass a **nullable** id to a function, or to split two simple functions including a non-nullable id and a non-param function?

Reply:
I think the split option is better in the relational database context.

However, CoreData is `NOT` a relational database, there is no primary key concept in CoreData, you can regard the `id` column as other regular columns, so I temporarily designed the function signature like that... 😅